apis/python/test/common.py

```diff
@@ -363,3 +364,31 @@ def quantize_embeddings_int8(
     starts = ranges[0, :]
     steps = (ranges[1, :] - ranges[0, :]) / 255
     return ((embeddings - starts) / steps - 128).astype(np.int8)
+
+
+def setUpCloudToken():
+    token = os.getenv("TILEDB_REST_TOKEN")
+    if os.getenv("TILEDB_CLOUD_HELPER_VAR"):
+        token = os.getenv("TILEDB_CLOUD_HELPER_VAR")
+    tiledb.cloud.login(token=token)
+
+
+def create_cloud_uri(name):
+    namespace, storage_path, _ = groups._default_ns_path_cred()
+    storage_path = storage_path.replace("//", "/").replace("/", "//", 1)
+    rand_name = random_name("vector_search")
+    test_path = f"tiledb://{namespace}/{storage_path}/{rand_name}"
+    return f"{test_path}/{name}"
+
+
+def delete_uri(uri, config):
```

Comment:
(Minor)(Optional)(Could be a future PR) Maybe we could move this into `apis/python/src/tiledb/vector_search/utils.py` and have `def delete_index(uri, config):` call it? Would avoid duplicate code.